core/arch/riscv64/riscv64.asm

```diff
@@ -233,7 +233,10 @@ GLOBAL_LABEL(dr_longjmp:)
         /* int atomic_swap(int *adr, int val) */
```

Comment:
Re the PR title:
s/Implemented atmoic_swap/Implemented atomic_swap/

Reply:
Thanks.